clients/drcachesim/tracer/instr_counter.cpp

```diff
@@ -41,6 +41,7 @@
 #include <stddef.h>
```

Comment:
Before submitting, maybe we should test this on a large app to make sure everything is WAI.